.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -1,10 +1,10 @@
 name: Feature request
 description: Suggest an idea for this project
-title: 'auro-formvalidation: [feature summary]'
+title: 'wc-generator: [feature summary]'
 labels:
   - 'Type: Feature'
   - not-reviewed
-  - 'auro-formvalidation'
+  - wc-generator
```

Comment:
revert

Reply:
Why would this repo title the tickets as though they were part of the wc-generator? This is not a pattern we follow anywhere else other than the actual generator.